config/routes.rb

```diff
@@ -20,11 +20,16 @@
       resources :opportunities
       resources :users
 
+      resources :lists, only: [:index] do
+        get :subscribe
+        get :unsubscribe
+      end
+
     end
   end
   get 'login', to: 'login#new'
   post 'login', to: 'login#create'
   delete 'logout', to: 'login#destroy'
 
-  get 'error', to: "errors#unauthorized"
+  get 'error', to: 'errors#unauthorized'
```

Comment:
does this controller action still exist? I don't see it.

Reply:
it's here: https://github.com/openstax/openstax-salesforce-api/blob/master/app/controllers/errors_controller.rb#L6